rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/activity/CovidPatientHistoryActivity.java

```diff
@@ -42,6 +43,11 @@ protected void onCreate(Bundle savedInstanceState) {
 
         currPatient = getIntent().getParcelableExtra(Constants.FormFields.PATIENT);
         presenter = new CovidPatientHistoryActivityPresenter(this);
+        patientVisitDate = getIntent().getStringExtra(Constants.FormFields.PATIENT_VISIT_DATE);
+
+        TextView tv = findViewById(R.id.visit_date_label);
+        String visitDateLabel = getString(R.string.visit_date_label);
+        tv.setText(visitDateLabel + ": " + patientVisitDate);
```

Comment:
Could this move to `populatePatientHistory()` function?

Reply:
Sure.